rama-http/src/service/fs/serve_dir/tests.rs

```diff
@@ -33,6 +33,26 @@ async fn basic() {
     assert_eq!(body, contents);
 }
 
+#[tokio::test]
+async fn basic_embedded() {
```

Comment:
same same here, try to avoid all the duplication. From a user perspective it's pretty much the same so would be good if at the very least every test is defined just once but ran for both embedded and non-embedded

Reply:
I’ve reused the shared parts of theese test cases. Also, because include_dir has performance issues when pointed at the repository root, I changed some cases to use the README.md and Cargo.toml in the rama-http directory.